### PR TITLE
{data}[GCCcore/10.2.0] XML-LibXML v2.0206

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — DISMISSED** [review]
I don't think this belongs to GCC. Should stay in GCCcore

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node354.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/935662792bd953950e890a7e4232a100 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node371.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/78af8535e4a1b45ffb7e480e0ca42594 for a full test report.

File: easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0206-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+name = 'XML-LibXML'
+version = '2.0206'
+
+homepage = 'https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod'
+description = "Perl binding for libxml2"
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
```
Comment (lexming) [inline easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0206-GCCcore-10.2.0.eb:12]: May I ask why did you add `pkg-config`? I do not see it in the build log. 
```suggestion
```

**Review by lexming — COMMENTED** [review]
The build works well, I just have a minor suggestion/question. Please see below

File: easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0206-GCCcore-10.2.0.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0206-GCCcore-10.2.0.eb:12]: Ubuntu does not have pkg-config.  The build fails without pkg-config.
```eb XML-LibXML-2.0206-GCCcore.eb --rebuild
== temporary log file in case of crash /tmp/eb-RxAuVe/easybuild-o4oLky.log
== found valid index for /app/software/EasyBuild/4.3.2/easybuild/easyconfigs, so using it...
== found valid index for /app/software/EasyBuild/4.3.2/easybuild/easyconfigs, so using it...
== processing EasyBuild easyconfig /app/build/XML-LibXML-2.0206-GCCcore.eb
== building and installing XML-LibXML/2.0206-GCCcore-10.2.0...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== taking care of extensions...
== installing extension File::chdir 0.1010 (1/4)...
== installing extension Alien::Base 2.38 (2/4)...
== installing extension Alien::Libxml2 0.17 (3/4)...
== FAILED: Installation ended unsuccessfully (build directory: /app/build/XMLLibXML/2.0206/GCCcore-10.2.0): build failed (first 300 chars): cmd " perl Makefile.PL PREFIX=/app/software/XML-LibXML/2.0206-GCCcore-10.2.0 " exited with exit code 2 and output:
Could not find an appropriate pkg-config or pkgconf implementation, please install PkgConfig.pm, PkgConfig::LibPkgConf, pkg-config or pkgconf at /app/build/XMLLibXML/2.0206/GCCcore-10.2 (took 16 sec)

Comment (lexming) [reply easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0206-GCCcore-10.2.0.eb:12]: Interesting, so it does use `pkg-config` and it can use it directly without the Perl module `PkgConfig`. Thanks :+1:

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @fizwit !
